src/forms/RadioButtonGroup.jsx

```diff
@@ -84,13 +60,6 @@ RadioButtonGroup.propTypes = {
 	/** Additional class name/s to add to the wrapper element */
 	className: PropTypes.string,
 
-	/** The axis the radio buttons will be layed out on */
-	direction: PropTypes.oneOf(['row', 'column']),
```

Comment:
@hsbacot I removed the `direction` prop because it relies on `Flex`. Is this correct? If not, what should be the correct behavior here?

Reply:
We need to support this since it used to determine the layout of the radio buttons in some places (and is actively used in mup-web). We could add support via css that is conditionally applied here or return the flex structure while in mwc since the flex shouldn't impact the styles.

Reply:
My only thought on this is that if at some point in time we want product teams to move from using MWC -> Swarm-Components, they're going to need to implement the CSS to create a column layout because we don't intend to provide this to them. So either 1) we could make this change now and write the css in the mup-web integration branch (not in this branch), or 2) we keep Flex but defer it to later for product teams. 

I can add Flex back in, but I just think this component needs a bit more planning after the first release. I'm also leaning toward creating a `RadioButtonGroup` component in `mup-web` that does state management + css/layout with data selectors because ultimately that's what's going to HAVE to happen when we remove MWC and are solely using Swarm-Components. We shouldn't end up in a state where every product team creates their own, new RadioButtonGroup component each time, which is what will happen unless we plan / influence it somehow.

Reply:
We should keep support for flex here, it will be around in some form for an indefinite amount of time. It is still being decided how spacing should work as swarm evolves.

Reply:
Functionally we should document how to handle state for a radio group but we should not handle it because that would result in duplicate state being stored.